server/schedulers/hot_region.go

```diff
@@ -658,11 +533,11 @@ func (bs *balanceSolver) filterSrcStores() map[uint64]*storeLoadDetail {
 	return ret
 }
 
-func (bs *balanceSolver) checkSrcByDimPriorityAndTolerance(minLoad, expectLoad *storeLoad) bool {
+func (bs *balanceSolver) checkSrcByDimPriorityAndTolerance(minLoad, expectLoad *storeLoad, toleranceRatio float64) bool {
```

Comment:
The tolerance in https://github.com/tikv/pd/pull/3900/files#diff-60d63e890af612b2f98b35d429cb583fc1143473e1b920bc0a57812d787905bcR545 should also be modified.